Realm/RLMObjectSchema.mm

```diff
@@ -276,6 +276,7 @@ - (void)setTable:(realm::Table *)table {
         p.object_type = prop.objectClassName ? prop.objectClassName.UTF8String : "";
         p.is_indexed = prop.indexed;
         p.is_primary = (prop == _primaryKeyProperty);
+        p.table_column = prop.column;
```

Comment:
sure, for now it does nothing, but it does more correctly create the `objectStoreCopy` if you ever were to use a fully-initialized `RLMProperty` as the source